examples/play-create-dv.yml

```diff
@@ -20,7 +20,7 @@
                   url: docker://quay.io/containerdisks/fedora:latest
               storage:
                 accessModes:
-                - ReadWriteOnce
+                  - ReadWriteOnce
```

Comment:
Drop this change?

Reply:
yamllint complains about zero indent lists, sorry, have to keep it

Reply:
What about other example yamls, do they have to be updated too?

Reply:
oh yes, sorry I missed a couple

Reply:
strange they weren't picked up by ansible-lint

Reply:
Added ansible-lint to the CI, fixed all linter errors